CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ and this project adheres to
 
 ## [unreleased]
 
+### Added
+
+- ​Added `opam-repositories` input to support multiple opam repositories.
+
+### Removed
+
+- The `opam-repository` input has been removed in order to add the
+  `opam-repository` input.
```

Comment:
```suggestion
- The `opam-repository` input has been removed in order to add the
  `opam-repositories` input.
```
Might it be worth supporting both inputs, but having a workflow error if you give both?

Reply:
Indeed, during the alpha release, we can support both. And now I feel that it's better to stop using the plural form for the input and instead use the singular form. (It's currently being decided in a rather vague sense.)

Reply:
The plural seems better to me if you actually have more than one!